client/pdo/client/scripts/EServiceDatabaseCLI.py

```diff
@@ -0,0 +1,299 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+import hashlib
+import os
+import sys
+
+import pdo.common.config as pconfig
+import pdo.common.logger as plogger
+
+import pdo.service_client.service_data.eservice as eservice_db
+
+import logging
+logger = logging.getLogger(__name__)
+
+# -----------------------------------------------------------------
+# -----------------------------------------------------------------
+def _hashed_identity_(enclave_id) :
+    return hashlib.sha256(enclave_id.encode('utf8')).hexdigest()[:16]
+
+# -----------------------------------------------------------------
+# -----------------------------------------------------------------
+def _load_database_(config, create=False) :
+    eservice_db.clear_all_data()
+    database_file = config['Service']['EnclaveServiceDatabaseFile']
+
+    if create and not os.path.exists(database_file) :
+        _save_database_(config)
+
+    try :
+        eservice_db.load_database(database_file, merge = False)
+    except Exception as e :
```

Comment:
the eservice_db.load_database method returns False instead of raising exception during load failure. This return value must be checked here inorder to raise an exception

Reply:
fixed.